vitest.config.js

```diff
@@ -0,0 +1,23 @@
+// Disabling this because ESLint is having problems resolving this file,
+// as long as vitest runs this isn't a real issue
+// eslint-disable-next-line import/no-unresolved
+import { defineConfig } from 'vitest/config'
+import graphql from '@rollup/plugin-graphql'
+
+export default defineConfig({
+  test: {
+    globals: true,
+    setupFiles: 'test-env.js',
+    coverage: {
+      enabled: true,
+      exclude: [
+        '**/handler.js'
```

Comment:
It does exclude that file. Each test creates its own instance of ApolloServer, which is mostly all the handler does. There might be a way to test that file better, but we haven't spent the time to do that. This setting was copied from the previous setup, so the coverage report stays consistent.

Reply:
The only work that the handler does that isn't an external library is the acceptance of the authentication headers, that _could_ be pulled into a util or something to make it testable. The important part of this is that the code is actually tested because it runs in many tests, its just the coverage is ignored.